src/js/calculate.js

```diff
@@ -0,0 +1,133 @@
+import operate from './operate';
```

Comment:
- [ ] **[OPTIONAL]** I think you should put this file and `operate.js` in a `logic` folder as the name `logic` makes more sense than `js`, which is too generic. What do you think?

Reply:
@TedLivist I appreciate your feedback, and yes, it for sure makes more sense! thank you.